sdc.py

```diff
@@ -48,9 +48,10 @@
 
 class StreamsetsDataCollector:
 
-    def __init__(self, version, namespace, registry):
-        self._image_name = IMAGE_NAME_TEMPLATE.format(registry, namespace, version)
-        self.image_name_no_registry = IMAGE_NAME_NO_REGISTRY_TEMPLATE.format(namespace, version)
+    def __init__(self, version, namespace, registry, os_major_version):
+        self.os_major_version = os_major_version
+        self._image_name = IMAGE_NAME_TEMPLATE.format(registry, namespace, version, os_major_version)
```

Comment:
How come `_image_name` is a "private" thing but `image_name_no_registry` isn't? The naming is a bit confusing. We may also want to simplify things if we find the need to treat "Image name" and "Image name no registry" as separate things (e.g. maybe just refer to one as "full_image_name" and one as "image_name" where we note that the latter includes registry details? Just a thought.

Reply:
Done.